sorc/ncep_post.fd/grib2_module.f

```diff
@@ -751,6 +751,16 @@ subroutine gengrb2msg(idisc,icatg, iparm,nprm,nlvl,fldlvl1,fldlvl2,ntrange,tinvs
          scale_fct_fixed_sfc2=0
        endif
 
+       ! Sending an empty key string to g2tmpl is ALWAYS an error. Yet, the post does this for many fields.
+       ! Fixing that requires refactoring post GRIB2 code and xml reader. This is a workaround for one
+       ! problematic case of the fixed_sfc2_type that generates numerous error messages in g2tmpl 1.12.0
+       if(len_trim(fixed_sfc2_type) == 0) then
+         ! Internally, due to a g2tmpl bug, when fixed_sfc2_type is invalid, it ends up with the same
+         ! value as fixed_sfc1_type. This assignment produces that effect without an error message.
+         fixed_sfc2_type = pset%param(nprm)%fixed_sfc1_type
```

Comment:
@SamuelTrahanNOAA From my testing, the fixes are:
```
fixed_sfc2_type = 'missing'
pset%param(nprm)%fixed_sfc2_type = 'missing'
```

Reply:
Please use the github "suggested change" feature, and I'll apply the change.